packages/preview/tauthesis/0.3.2/typst.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "tauthesis"
+version = "0.3.2"
+entrypoint = "template/main.typ"
+authors = ["Santtu Söderholm <santtu.soderholm@tuni.fi>"]
+license = "MIT"
+description = "A template for writing Tampere University theses."
+homepage = "https://gitlab.com/tuni-official/thesis-templates/tau-typst-thesis-template"
+repository = "https://gitlab.com/tuni-official/thesis-templates/tau-typst-thesis-template"
+keywords = ["Tampere University", "thesis template"]
+categories = ["thesis", "layout"]
```

Comment:
I think that the `layout` category should be dropped

Reply:
This was resolved by 73ec8f60b994513da64f4fd1102c3d7d0e0361dd.